src/pages/home/report/ReactionList/PopoverReactionList.js

```diff
@@ -184,7 +184,7 @@ class PopoverReactionList extends React.Component {
             };
         }
         const emojiCount = selectedReaction.users.length;
-        const reactionUsers = _.map(selectedReaction.users, (sender) => sender.accountID.toString());
+        const reactionUsers = _.map(selectedReaction.users, (sender) => sender.accountID);
         const emoji = _.find(emojis, (e) => e.name === selectedReaction.emoji);
         const emojiCodes = EmojiUtils.getUniqueEmojiCodes(emoji, selectedReaction.users);
         const hasUserReacted = Report.hasAccountIDReacted(this.props.currentUserPersonalDetails.accountID, reactionUsers);
```

Comment:
Oofda - `Report.hasAccountIDReacted` is checking that accountIDs equal each other by converting accountIDs to strings (probably why we had `.toString()` above before - can we update that function to treat `userAccountID`s as numbers? 🙏 

Unless you think that has the potential to cause bigger problems...

Reply:
Done, I think this also fixed the tooltips conten